main.py

```diff
@@ -27,24 +27,28 @@
 from commands.restart import restart
 from commands.gothefucktosleep import gothefucktosleep
 from commands.boba import boba
+
 from commands.confessions import confession
 from presence import watching
 import bot
 import sys
+import utils
 from git import Repo
 
 
 # load environment vars (from .env)
 load_dotenv()
 prod = len(sys.argv) > 1 and sys.argv[1] == "prod"
 TOKEN = os.getenv("DISCORD_TOKEN_DEV" if not prod else "DISCORD_TOKEN")
-GUILDS = os.getenv("DISCORD_GUILD").split(",")
+GUILDS_DICT = utils.read_json_file("guilds.env.json")

```

Comment:
Ah for the guilds dict I was thinking of something like:
```json
{
  "[TW_SERVER_ID]": {
    "name": "taiwan",
    "confessions_channel": [TW_SERVER_CONFESSIONS_CHANNEL_ID]
  },
  "[BALTICS_SERVER_ID]": {
    "name": "baltics",
    "confessions_channel": [BALTICS_SERVER_CONFESSIONS_CHANNEL_ID]
  },
}
```

And then you can just feed the keys into the register_commands for the confessions by doing `guilds=[discord.Object(id=id) for id in GUILDS_DICT.keys()]` or smth like that, and you can get stuff like the name of the server by doing `GUILDS_DICT[id].name`

Reply:
So you can't say like "this command should only exist in the TW and Baltics servers specifically" without looping through the dictionary